### ISSUE TITLE
[Accessibility issue] Invalid input to your email text box and click on subscribe Voiceover does not announce the error message correctly

### ISSUE THREAD

**Issue by harshamulpuri2913** [ampproject/amphtml#3889]
## Short description of your issue: within iOS device (IPAD / IPOD), entering invalid format of email as input to your email textbox voiceover does not read out valid error message

Test URL: https://amp-mk-1.herokuapp.com/examples.build/forms.amp.max.html

Pre-requisite: Open IPAD--> Settings --> General --> Accessibility --> Voiceover , Set to ON
## How do we reproduce the issue?
1. Logon to IPAD and open the safari browser
2. Open test URL provided above
3. Under "Subscribe to our weekly newsletter (xhr)", please click on subscribe button without by entering invalid format of your email address. 

Actual results: 
1. Voiceover reads out as "your email text field is editing character mode required insertion point at start"

Expected result: 
1. Should read out as "email format entered is invalid / mismatch" or a more appropriate error message
## What browsers are affected? Safari , Chrome

All browsers? Some specific browser? What device type?
Device details: IPAD 
iOS version: 9.3.2 (13F69)

Attachments: 
![img_0044](https://cloud.githubusercontent.com/assets/20247457/16535672/21e024a0-3f9e-11e6-955e-c14dd05cdd2e.PNG)

**Comment by adelinamart** [comment]
@rudygalfi do we still need this?

**Comment by jridgewell** [comment]
I think @mkhatib addressed this?

**Comment by adelinamart** [comment]
@harshamulpuri2913 can you try reproducing this, and if it's still an issue, please re-open this. Thanks.
